### PR TITLE
Mozorg and Pocket Demos on Cloud Run

### PR DESCRIPTION
**Pull request by stevejalim**
This changeset contains the code that's needed to support running demos on GCP Cloud Run.

It has been developed against my GCP sandbox, and so there are SRE/ops-level changes that also need to be made before we can roll this out into use.

Merging to `main` in the meantime will not break anything.

**For context and before reviewing, please see [the parent Issue](https://github.com/mozilla/bedrock/issues/11641) which outlines the overall approach, plus contains a checklist of the other SRE/devops tasks that need to happen to make all this work.**

Example live sites (without custom domains)
* ~https://pocket-demo-1-vfex4a7vcq-uc.a.run.app/en/about/~ currently offline
* ~https://mozorg-demo-3-vfex4a7vcq-uc.a.run.app/en-GB/~ currently offline

### CODE REVIEW

File: gcp/bedrock-demos/cloudbuild/demo-mozorg.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ]
+
+  - name: 'gcr.io/google.com/cloudsdktool/cloud-sdk'
+    entrypoint: 'gcloud'
+    args:
+      - 'run'
+      - 'deploy'
+      - '${BRANCH_NAME}'  # Branch name is the Service name
+      - '--image=us-central1-docker.pkg.dev/${PROJECT_ID}/bedrock-demo/${BRANCH_NAME}'  # Branch name is image name
+      - '--env-vars-file=/workspace/gcp/bedrock-demos/cloudrun/mozorg-demo.env.yaml' # Mozorg-specific demo env vars
```
Comment (stevejalim) [inline gcp/bedrock-demos/cloudbuild/demo-mozorg.yaml:25]: /workspace/ is the dir in Cloud Build where bedrock is checked out. I can improve the comment to make that clear

File: gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+ALLOWED_HOSTS: .tekcopteg.org,.moz.works,.run.app
+CONTENT_CARDS_URL: https://www-dev.allizom.org/media/
+CSP_DEFAULT_SRC: "*.tekcopteg.com"
+CSP_EXTRA_FRAME_SRC: "*.mozaws.net,o1069899.sentry.io"
+CSP_REPORT_ENABLE: "True"
+DB_DOWNLOAD_IGNORE_GIT: "True"
+DEBUG: "False"
+DEV: "True"
+GTM_CONTAINER_ID: GTM-MW3R8V
```
Comment (alexgibson) [inline gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml:15]: This is mozorg's GTM container ID, is it needed for the Pocket deploy also?

File: gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+LOG_LEVEL: INFO
+PROD_DETAILS_STORAGE: product_details.storage.PDDatabaseStorage
+RUN_SUPERVISOR: "True"
+SECURE_SSL_REDIRECT: "True"
+SENTRY_DSN: https://97ec0cd426714b728e92f3b3aa62f00b@o1069899.ingest.sentry.io/6260338
```
Comment (alexgibson) [inline gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml:20]: Same question as above for Sentry?

File: gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+SITE_MODE: Pocket
+STUB_ATTRIBUTION_HMAC_KEY: the-dude
```
Comment (alexgibson) [inline gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml:22]: Same question as above for Stub attribution?

File: gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml (hunk 2 above)
Comment (stevejalim) [reply gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml:15]: Will check if needed but I will look for the pocket one. Thanks for the catch!

File: gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml (hunk 3 above)
Comment (stevejalim) [reply gcp/bedrock-demos/cloudrun/pocket-demo.env.yaml:20]: We currently have a generic "demos" sentry proj. We could split that too if there's enough value

File: gcp/bedrock-demos/cloudrun/mozorg-demo.env.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+CSP_REPORT_ENABLE: "True"
+DB_DOWNLOAD_IGNORE_GIT: "True"
+DEBUG: "False"
+DEV: "True"
+GTM_CONTAINER_ID: GTM-MW3R8V
+LOG_LEVEL: INFO
+PROD_DETAILS_STORAGE: product_details.storage.PDDatabaseStorage
+RUN_SUPERVISOR: "True"
+SECURE_SSL_REDIRECT: "True"
+SENTRY_DSN: https://97ec0cd426714b728e92f3b3aa62f00b@o1069899.ingest.sentry.io/6260338
```
[the remark this answers is not in the document]
Comment (pmac) [reply gcp/bedrock-demos/cloudrun/mozorg-demo.env.yaml:20]: oh, no worries at all. quite the contrary. really appreciate you looking out for us.

**Comment by stevejalim** [comment]
Hi @bkochendorfer and @pmac the initial Terraform config for setting this all up is over at https://github.com/mozmeao/infra-services/pull/162

The Cloud Build config originally from this PR is now over there and, as such, this PR now only contains the two demo-specific env-var files. Can I get an `r+` so I can merge this, please?

**Comment by stevejalim** [comment]
@alexgibson Could I get an `r+` when you have a mo? This changeset is now just public-appropriate env vars. Cheers!

**Pull request merged**